### ISSUE TITLE
Database chooser should work with Javascript disabled

### ISSUE THREAD

**Issue by Otterfan** [BCLibraries/bc-libraries-site#29]
Users without Javascript enabled should be able to access the database list. This could be partially (or fully) solved by resolving #3.
